Source/Android/app/build.gradle

```diff
@@ -89,3 +92,7 @@ dependencies {
     // Allows FRP-style asynchronous operations in Android.
     compile 'io.reactivex:rxandroid:1.2.1'
 }
+
+def getVersion() {
+    return 'git describe --abbrev=0'.execute([], project.rootDir).text.trim()
```

Comment:
I don't think we should be using `--abbrev=0` for this; otherwise every build would have the same version number. I don't know how this affects installation on a device tho.